python-checks/src/test/resources/checks/hotspots/clearTextProtocols.py

```diff
@@ -142,3 +142,62 @@ def FP_starttls_in_different_method():
 
 def start_tls(x):
   x.starttls(context=context)
+
+
+def python_web_server_noncompliant():
+    from http.server import SimpleHTTPRequestHandler, HTTPServer, ThreadingHTTPServer
+
+    http_server = HTTPServer(('0.0.0.0', 8080), SimpleHTTPRequestHandler)
+    http_server.serve_forever()  # Noncompliant
+#   ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    threading_http_server = ThreadingHTTPServer(('0.0.0.0', 8080), SimpleHTTPRequestHandler)
+    threading_http_server.serve_forever()  # Noncompliant
+
+    class MyServer(HTTPServer):
+        def run(self):
+            super(self).serve_forever()  # Noncompliant
+        #   ^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Maybe we should raise the issue on the entire function call, and not just the callee?